PaoProject.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
si acest fisier este de configurare de la intellij si trebuie pus in .gitignore si sa nu ii faci commit